learners/setup.md

```diff
@@ -2,53 +2,158 @@
 title: Setup
 ---
 
-FIXME: Setup instructions live in this document. Please specify the tools and
-the data sets the Learner needs to have installed.
+Vantage6 has hardware and software requirements in order to run the vantage6 node(s) and/or server. As for the detailed requirements, please refer to the [vantage6 documentation](https://docs.vantage6.ai/en/main/node/requirements.html).
 
-## Data Sets
+For now, we just need to focus on the software setup.
 
-<!--
-FIXME: place any data you want learners to use in `episodes/data` and then use
-       a relative link ( [data zip file](data/lesson-data.zip) ) to provide a
-       link to it, replacing the example.com link.
--->
-Download the [data zip file](https://example.com/FIXME) and unzip it to your Desktop
+## Software setup
 
-## Software Setup
+This lesson requires the following software to be installed on your system:
 
-::::::::::::::::::::::::::::::::::::::: discussion
+- Recommended operating system: Ubuntu 20.04+ , MacOS Big Sur+, or Windows 10+
+- Docker Desktop (always latest version)
+- Conda (latest version)
+- Python
+    - Python v3.10 for vantage6 version 3.8.0 or higher
```

Comment:
I would shorten this to just say "Python v3.10" (no-one just starting to use vantage6 should use versions below 3.8 :-))